docs/bonus-labs/cluster-autoscaling.md

```diff
@@ -0,0 +1,75 @@
+# Cluster Autoscaling
+
+Tanzu Kubernetes Grid supports cluster autoscaling leveragin the Cluster API provider.  Depending on your lab configuration, you may have enabled cluster autoscaling when your clusters were provisioned.
+
+## Lab Configuration Parameters
+
+You may have noticed the following keys in `params.yaml` that direct the use of cluster autoscaling for the shared services cluster and workload clsuter.
+
+General purpose information on how cluster autoscaler works can be found in the upstream [FAQ](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md).
+
+```yaml
+shared-services-cluster.autoscaler-enabled: true
+shared-services-cluster.worker-replicas: 2 # initial and minimum number of worker nodes
+shared-services-cluster.worker-replicas-max: 4 # maximum number of worker nodes
+workload-cluster.autoscaler-enabled: true
+workload-cluster.worker-replicas: 1
+workload-cluster.worker-replicas-max: 3
+```
+
+## Run the Exercises
+
+The following series of commands demonstrate cluster autoscaling...
+
+```bash
+
+DEMO_CLUSTER_NAME=$(yq e .shared-services-cluster.name $PARAMS_YAML)
```

Comment:
It's kind of obvious but maybe we want to be more explicit saying that we are doing the Autoscaling demo in the Shared Services Cluster.

Reply:
Done.